app/views/steps/_form_assets.html.erb

```diff
@@ -1,16 +1,25 @@
-<div class="panel panel-default panel-step-attachment">
-  <div class="panel-heading">
-    <span class="fas fa-paperclip"></span>
-    <%= t("protocols.steps.new.asset_panel_title") %>
-    <div class="pull-right">
-      <% unless ff.object.file.exists? && ff.object.locked? %>
-        <%= ff.remove_nested_fields_link do %>
-          <span class="fas fa-times"></span>
-        <% end %>
-      <% end %>
-    </div>
+<% asset = ff.object %>
+<% image_preview = asset.is_image? ? asset.url(:medium) : nil %>
+
+<div class="attachment-placeholder pull-left">
+  <div class="attachment-thumbnail <%= image_preview ? '' : 'no-shadow' %>">
+    <% if image_preview %>
+      <%= image_tag image_preview %>
+    <% else %>
+      <i class="fas fa-image"></i>
+    <% end %>
+  </div>
+  <div class="attachment-label"><%= truncate(asset.file_file_name,
+                                             length: Constants::FILENAME_TRUNCATION_LENGTH) %></div>
+  <div class="spencer-bonnet-modif">
+    Modified: <%= l(asset.updated_at, format: :full_date) if asset.updated_at %>  <br>
```

Comment:
This should also be in I18.t. With DRYing up above I meant to put this code
```
    <div class="attachment-thumbnail <%= image_preview ? '' : 'no-shadow' %>">
      <% if image_preview %>
        <%= image_tag image_preview %>
      <% else %>
        <i class="fas <%= file_fa_icon_class_v2(asset) %>"></i>
      <% end %>
    </div>
    <div class="attachment-label"><%= truncate(asset.file_file_name,
                                               length: Constants::FILENAME_TRUNCATION_LENGTH) %></div>
    <div class="spencer-bonnet-modif">
      <%= t('protocols.steps.attachments.modified_label') %> <%= l(asset.updated_at, format: :full_date) %>  <br>
      <%= number_to_human_size(asset.file.size) %>
    </div>
```
in a separate partial, since you already see problem with maintaining both piece of code.

Reply:
👍